### DISCUSSION TITLE
what does export as markdown format mean?

### DISCUSSION THREAD

**Discussion by zillionare** [slidevjs/slidev#1768] [Q&A]
Seems no documentation covered this in detail. Say, if I have a components <Card />, will this to be rendered as a picture then inserted back into the markdown? If not, what's the behaviour here?

**Comment by kermanx** [comment]
It will always be rendered as images and the Markdown will just be a collection of images🥲

**Reply by zillionare** [reply]
This is a common scenario when I use Slidev: I often write some posts, enlivening my articles with admonitions and custom components. When I publish my articles, most markdown-supporting websites do not support admonitions, let alone these components, so I need to manually convert these parts into images. If I were to implement these functionalities myself, how should I generally start?

**Reply by kermanx** [reply]
You can use the `@slidev/parser` package to divide the Markdown into slides, and then concat each slide's content (i.e., remove the frontmatters).
